review_checker/views/android_clients.view.lkml

```diff
@@ -0,0 +1,107 @@
+include: "//looker-hub/review_checker/views/android_clients.view.lkml"
+
+view: +android_clients {
+
+  dimension: client_id {
+    #primary_key: yes
+    sql: ${TABLE}.client_id ;;
+    hidden: yes
+  }
+
+  dimension: client_id_date {
+    primary_key: yes
+    sql: CONCAT(${TABLE}.client_id, ${TABLE}.submission_date) ;;
+    type: string
+  }
+
+  #measures
+  measure: active_hours {
+    type: sum
+    sql:  ${active_hours_sum};;
+    value_format: "0.##"
+  }
+
+  measure: active_hours_per_client {
+    type: number
+    sql: ${active_hours}/${client_count} ;;
+    value_format: "0.##"
+  }
+
+  measure: ad_clicks {
+    type: sum
+    sql: ${ad_click} ;;
+  }
+
+  measure: ad_clicks_rate {
+    type: number
+    sql: ${ad_clicks}/${client_count} ;;
+    value_format: "0.##"
+  }
+
+  measure: fx_dau {
+    type: sum
+    sql: ${is_fx_dau} ;;
+  }
+
+  measure: fx_dau_percentage {
+    type: number
+    sql: 100 * ${fx_dau}/${client_count} ;;
+    value_format: "0\%"
```

Comment:
I'd normally format as percentage instead of multiplying by 100 and forcing the % symbol. I think this would avoid confusion if creating additional derived calculations.
You can do so with `value_format_name: percent_2`